pandas/tseries/tests/test_resample.py

```diff
@@ -741,15 +741,15 @@ def test_resample_empty_dataframe(self):
 
         for freq in ['M', 'D', 'H']:
             # count retains dimensions too
-            methods = downsample_methods + ['count']
+            methods = downsample_methods + upsample_methods
             for method in methods:
                 result = getattr(f.resample(freq), method)()
 
-                expected = f.copy()
+                expected = pd.Series([])
```

Comment:
right, you are resampling an empty dataframe here, so the result must be an empty dataframe with the new freq.

its not a Series at all.